heron/packing/src/java/com/twitter/heron/packing/utils/RamRequirement.java

```diff
@@ -0,0 +1,65 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.twitter.heron.packing.utils;
```

Comment:
Typically we don't put model objects in a util package. Could we put the elsewhere and only have static utility helpers under util? A class representing RamRequirement seems like a core object, not a utility.